PROCESS.md

```diff
@@ -0,0 +1,279 @@
+
+The Haskell Security Response Team (HSRT) is responsible for coordinating
+the progressive disclosure of a vulnerability.
+
+Members of the team are independent and security-minded folks who ensure
+that vulnerabilities are dealt with in a timely manner and that
+downstream stakeholders are notified in a coordinated and fair manner.
+Where a member of the team is employed by a downstream stakeholder, the
+member does not give their employer prior notice of any vulnerabilities.
+In order to reduce the disclosure of vulnerability in the early stages,
+membership of this team is intentionally limited to a small number of
+people.
+
+## Supported versions
+
+The Vulnerability Management team coordinates patches fixing
+vulnerabilities in packages published via Hackage.
+
+## Process
+
+Each security bug is assigned a HSRT *coordinator* (member from the
+haskell security response team) that will drive the fixing and
+disclosure process. Here are the steps we follow.
+
+### Reception
+
+Private report can be received by email to [security-advisories@haskell.org](mailto:security-advisories@haskell.org),
+or via a [private issue on the security-advisories repo](https://github.com/haskell/security-advisories/security/advisories/new).
+
+Public report can be submitted as a [regular issue on the security-advisories repo](https://github.com/haskell/security-advisories/issues).
+
+The first steps performed by the HSRT are to:
+
+-   create a bug if one does not yet exist.
+-   check that the report indicates the correct project and adjust as
+    necessary.
+-   contact the project's maintainer for confirmation
+    of impact and determination of affected version.
+
+### Patch development
+
+For a private report, the reporter (automatic if reported directly as a
+bug) and the affected projects' maintainers plus anyone they deem necessary
+to develop and validate a fix are added to the bug's notified list.
+A fix is proposed as a patch to the private bug report,
+**not sent to the public code review system**.
+
+For public reports, there is no need to directly subscribe anyone and
+patches can be submitted directly to the code review system instead of
+as bug attachments (though the bug should be referenced in any commit
+messages so it will be updated automatically).
+
+If project-side delays are encountered at this or any subsequent stage
+of the process, the HSRT and other interested parties may reach out to
+that project's maintainers requesting more immediate attention to the issue.
+
+### Patch review
+
+For a private report once the initial patch has been attached to the
+bug, core reviewers on the subscription list from the project in
+question should review it and suggest updates or pre-approve it for
+merging. Privately-developed patches need to be pre-approved so that
+they can be fast-tracked through public code review later at disclosure
+time.
+
+### Draft advisory
+
+In the mean time, the HSRT coordinator prepares a security advisory
```

Comment:
I think that we should work with the person who reports it to refine their draft security advisory, rather than writing them ourselves.